Update serverless_tracing_and_webpack.md (#16155)
DataDog/documentation: content/en/serverless/guide/serverless_tracing_and_webpack.md

    };
    ```

> 4. If you are using the `serverless-webpack` and have the option `includeModules` set to any value other than `false`, serverless-webpack automatically [packs external moduels under node_modules][5]. Therefore we need to force exclude `datadog-lambda-js` and `dd-trace`. Skip this step if you don't use `serverless-webpack` or you don't see the option `includeModules` in your serverless.yml.

Reviewer: ```suggestion
4. If you are using the `serverless-webpack` and have the option `includeModules` set to any value other than `false`, serverless-webpack automatically [packs external modules under node_modules][5]. Therefore, you must force exclude `datadog-lambda-js` and `dd-trace`. Skip this step if you don't use `serverless-webpack` or you don't have the `includeModules` option in your serverless.yml.
```
```suggestion
4. If you are using the `serverless-webpack` and have the option `includeModules` set to any value other than `false`, serverless-webpack automatically [packs external moduels under node_modules][5]. Therefore we need to force exclude `datadog-lambda-js` and `dd-trace`. Skip this step if you don't use `serverless-webpack` or you don't see the option `includeModules` in your serverless.yml.
```

    **serverless.yml**

    ```
    custom:
      webpack:

>         # You only need everything below if you already have the includeModules option configured

Reviewer: ```suggestion
        # Note: You only need the following if you already have the includeModules option configured
```
```suggestion
        # You only need everything below if you already have the includeModules option configured
```
